server/middleware/get-contract-by-id-from-param.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const getContractByID = require('../lib/get-contract-by-id');
+
+
+module.exports = (req, res, next) => {
+	const {contract_id} = req.params || {};
+	if (!contract_id) {
+		return res.sendStatus(400);
+	}
+	getContractByID(contract_id)
```

Comment:
does this need to be returned or awaited?

Reply:
Nope, not in a synchronous function.

Reply:
But `getContractByID` isn’t synchronous is it?